src/components/formSections/AmendmentFormInfo.js

```diff
@@ -100,7 +100,11 @@ class AmendmentFormInfo extends React.Component {
             />
           </Col>
           <Col style={{ paddingTop: 20 }} xs={2}>
-            <Field name="openEnded" type="checkbox">
+            <Field
+              defaultValue={false}
```

Comment:
Passing a _defaultValue_ prop to a Field _dirties_ the Field and in turn enables the Save and Close on initial render. Instead, lets tackle this by adding an `openEnded: false` entry to the initialValues object created in the _CreateAmendmentRoute_ file. That way the submit button stays disabled initially.